sympy/ntheory/factor_.py

```diff
@@ -1886,6 +1893,12 @@ class udivisor_sigma(Function):
 
     divisor_count, totient, divisors, udivisors, udivisor_count, divisor_sigma,
     factorint
+
+    References
+    ==========
+
+    .. [1] http://mathworld.wolfram.com/UnitaryDivisorFunction.html
```

Comment:
There is a similar reference in the docstring of the method `udivisor_count`, which could be improved along with the addition of `Examples` header.

Reply:
Gah! I thought I did it (somewhat) systematically now (although admittedly quick...).